application/dash_application/views/call_ready_rna.py

```diff
@@ -66,7 +66,8 @@ def dataversion():
     "Total Reads (Passed Filter)": "Total reads passed filter",
```

Comment:
Is the Total Reads special_col still required?

Reply:
This was both reads and cluster metrics will be in the table.